src/hooks/useToast.tsx

```
@@ -0,0 +1,97 @@
+                return updated;
+            });
+        },
+        []
+    );
+
+    const removeToast = useCallback((id: number | string) => {
+        setToasts((prev) => prev.filter((t) => t.id !== id));
+    }, []);
+
+    const ToastPortal = (
+        <>
```

ESLint

[error] 'React' must be in scope when using JSX

(react/react-in-jsx-scope)